the-example-app/README.md

```diff
@@ -1,36 +1,31 @@
-What is `The Example App model`?
+Content model: `The Example App`
 ================================
 
 The example app content model containts the model for all the example apps, demonstrating in a broader fashion some examples on how to use
-Contentful. If you want to see the model in action, we do have a [hosted version](https://contentful-example-app.herokuapp.com/), for you
+Contentful. If you want to see the model in action, we do have a [hosted version](https://the-example-app-nodejs.herokuapp.com/), for you
 to check out.
 
-Through our [command line app](https://github.com/contentful/contentful-cli) you can push this model to your own space. Doing so will be
+Through our [command line app](https://github.com/contentful/contentful-cli) you can import this model into your own space. Doing so will be
```

Comment:
I would consider removing the second sentence entirely. Just leaving:
"Through our [command line app](https://github.com/contentful/contentful-cli) you can import this model into your own space."

and then following with the command.